### PR TITLE
USWDS - contribution guide: Add basic development instructions

### LINKED ISSUE #6003: Add development instructions to contribution guide
The contribution guide at https://github.com/uswds/uswds/blob/develop/CONTRIBUTING.md does not provide any details about how to develop USWDS, e.g. run locally, run tests, or format code. This would be useful to developers looking to submit a pull request.

### PR DESCRIPTION
**Pull request by jeffpw-goog**
# Summary

Add basic development environment instructions to the contribution guide, so that developers can get further on a pull request without needing to ask for help.

## Breaking change

This is not a breaking change.

This changes docs only.

## Related issue

Closes #6003

## Problem statement

The contribution guide for submitting a pull request skips from "create a branch" to "create a pull request" with no instructions for how to set up a development environment, run tests, format code, etc.

## Solution

Add instructions to cover the above information.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,9 +74,11 @@ If you want to propose a code fix, even better! Here’s a few steps to help you
...
 4. Fork this repo into your GitHub account. Read more about forking a repo on [GitHub](https://help.github.com/articles/fork-a-repo/). 
 5. Create a branch from `develop` and name it in a way that lightly defines what you’re working on (for example, `add-styles`).
-7. Once you’re ready to submit a pull request, fill out the [pull request template](https://github.com/uswds/uswds/compare).
-8. Link your pull request to the issue you created. This important step helps us know which issue this solution fixes. Tip: You can link the pull request in the body of the pull request template using the GitHub comment `closes #issue-no` or `resolves #issue-no`. You can read more about linking pull requests on [GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
-9. Submit your pull request against the `develop` branch. 
+6. Run `npm install` to build and then `npm start` to run locally. Check `package.json` for available development commands, e.g. `npm run prettier` to format the code.
+7. Run `gulp test` to run tests (`npm test` includes `snyk` which requires extra configuration and will be run automatically on your pull request).
```
Comment (mejiaj) [inline CONTRIBUTING.md:78]: **issue:** This setup information could get easily lost. We should consider separating it into its own section.

**Review by mejiaj — COMMENTED** [review]
Thanks for the submission. This is a good addition to the docs, but concerned it could get easily lost. Plus, it's not specific to bug fixes.

@amyleadem @thisisdano thoughts on creating a separate section for local environment setup?

File: CONTRIBUTING.md (hunk 1 above)
Comment (amyleadem) [reply CONTRIBUTING.md:78]: Good suggestion, @mejiaj. I'll work on drafting an alternate version of this update with this guidance in its own section.

Comment (amyleadem) [reply CONTRIBUTING.md:78]: Update: I have opened up PR #6038 as an alternate solution for this issue.

Comment (amyleadem) [reply CONTRIBUTING.md:78]: Update: I have opened up PR #6038 as an alternate solution for this issue.

**Comment by amyleadem** [comment]
Thanks for submitting this PR, @jeffpw-goog. To address concerns about the structure of this content, I've opened PR #6038 as an alternate solution.

I am going to close this PR in favor of #6038. Please let us know if you have any questions or comments!

**Comment by jeffpw-goog** [comment]
Thanks @amyleadem!

**Pull request closed without merging**